LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -316,25 +323,39 @@ private struct CorrectionRangeOverridesGuardrailWarning: View {
     private func singularWarningTitle(for threshold: SafetyClassification.Threshold) -> Text {
         switch threshold {
         case .minimum, .belowRecommended:
-            return Text("Low Correction Value", comment: "Title text for the low correction value warning")
+            switch preset {
+            case .preMeal:
+                return Text("Low Pre-meal Value", comment: "Title text for the low pre-meal value warning")
+            case .workout:
+                return Text("Low Workout Value", comment: "Title text for the low workout value warning")
+            }
         case .aboveRecommended, .maximum:
-            return Text("High Correction Value", comment: "Title text for the high correction value warning")
+            switch preset {
+            case .preMeal:
+                return Text("High Pre-Meal Value", comment: "Title text for the low pre-meal value warning")
```

Comment:
[nit] consider being consistent with "Pre-Meal" vs "Pre-meal".  (I don't care which, just be consistent...)